alacritty_terminal/src/term/mod.rs

```diff
@@ -687,6 +687,55 @@ impl VisualBell {
     }
 }
 
+#[derive(Debug, Clone, Default)]
```

Comment:
This struct is cheap to copy, so adding Copy to it make sense. (don't forget to remove .clone() in codebase)